dbschema/project.esdl

```diff
@@ -174,13 +174,13 @@ module default {
           __new__.mouEnd,
           interval
         )
-      if __old__.financialReportPeriod != __new__.financialReportPeriod (
+      select (if __old__.financialReportPeriod ?!= __new__.financialReportPeriod then (
```

Comment:
@CarsonF  - I'm getting this error here.

![Window_and_project_esdl_—_cord-api-v3](https://github.com/user-attachments/assets/d54d9d6d-7cc8-4488-9f61-f9c2e8d8285e)

I'm trying to avoid `ProgressReport`s here since that's a different flow.  Any ideas on the best way to handle this?